code/modules/mob/living/say.dm

```diff
@@ -110,6 +110,10 @@ var/list/channel_to_radio_key = new
 			S.message = stutter(S.message)
 			verb = pick("stammers","stutters")
 			. = 1
+		if(reagents.has_reagent("hyperzine")) // motor mouthed hyperzine user

```

Comment:
```suggestion
		if(reagents?.has_reagent("hyperzine")) // motor mouthed hyperzine user
```
`reagents` is lazy

Also unfortunately this won't work for humans. We have a few different reagent holders, declared on /mob/living/carbon, such as `bloodstr` (the one you should check in this case, I imagine), `ingested`, and `touching`.

You'd have to put this in `carbon/handle_speech_problems()` or, add it to the existing `human/handle_speech_problems()` and read `bloodstr`.

Reply:
somehow, `reagents` on `carbon` actually does point towards `bloodstr`, but `ingested` and `touching` aren't covered, so fair point

i've got a better idea for this, though, now that i've realized that there was a `handle_speech_problems` for humans

Reply:
Ah, it sets reagents to reference bloodstr in carbon/Initialize(). That's `interesting`. Fair enough.